packages/loot-core/src/client/privacy.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { createContext, useContext } from 'react';
+import { useSelector } from 'react-redux';
+
+type TPrivacyModeContext = {
+  enabled: boolean;
+};
+
+let PrivacyContext = createContext<TPrivacyModeContext>(null);
+
+export function PrivacyProvider({ children }) {
+  let prefs = useSelector(state => state.prefs);
+  let isPrivacyEnabled = prefs
+    ? prefs.local
+      ? prefs.local.isPrivacyEnabled
+      : false
+    : false;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
  let isPrivacyEnabled = prefs?.local?.isPrivacyEnabled || false;
```